bindings/protocols/coap/index.html

```diff
@@ -1,181 +1,464 @@
 <!DOCTYPE html>
 <html>
+
 <head>
     <meta charset='utf-8'>
     <script src='https://www.w3.org/Tools/respec/respec-w3c' class='remove'></script>
     <script class='remove'>
         var respecConfig = {
             specStatus: "ED",
             editors: [{
-                name: "Michael Koster",
-                w3cid: "110658",
-                company: "SmartThings",
-                companyURL: "https://www.smartthings.com/"
-                },
-                {
-                name: "Ege Korkan",
-                w3cid: "110131",
+                name: "Klaus Hartke",
+                w3cid: "107859",
                 company: "Siemens AG",
                 companyURL: "https://www.siemens.com/"
-                }
-            ],
+            }],
             edDraftURI: "https://w3c.github.io/wot-binding-templates/bindings/protocols/coap/",
             shortName: "wot-coap-binding",
             github: "https://github.com/w3c/wot-binding-templates/tree/main/bindings/protocols/coap",
+            localBiblio: {
+                "I-D.ietf-core-new-block": {
+                    title: "Constrained Application Protocol (CoAP) Block-Wise Transfer Options Supporting Robust Transmission",
+                    href: "https://datatracker.ietf.org/doc/draft-ietf-core-new-block/14/",
+                    "status": "Internet Draft (work in progress)",
+                    "publisher": "IETF",
+                    "date": "14 February 2022",
+                    "authors": [
+                        "M. Boucadair",
+                        "J. Shallow"
+                    ],
+                },
+            },
         };
     </script>
-    <title>CoAP Binding Template</title>
+    <title>CoAP binding template</title>
```

Comment:
It would be better to keep capitilization, I have realized that it is wrong in Modbus now

Reply:
I've started off with index.template.html and just replaced `BINDINGNAME` with `CoAP`:

https://github.com/w3c/wot-binding-templates/blob/09de2a43d80188171c9d265de4d17d6444382066/bindings/index.template.html#L20

Will fix.

Reply:
BTW, why are these documents called *templates*? Would perhaps "CoAP Binding for Web of Things" be a better title?

Reply:
That is a valid discussion, also see https://github.com/w3c/wot-binding-templates/issues/143